source/quick-start.txt

```diff
@@ -37,11 +37,62 @@ to check your version of Node and NPM and installation instructions for your
 system, see `downloading and installing Node.js and npm
 <https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
-Initialize the Project
-~~~~~~~~~~~~~~~~~~~~~~
-
 After you verify that you installed the required versions of Node and NPM,
-create a new project.
+create a new project by :ref:`cloning our js-starter repository
+<qs-download-the-repo>` or by :ref:`creating it manually <qs-manual-setup>`.
+
+.. _qs-download-the-repo:
+
+Clone the Project
+~~~~~~~~~~~~~~~~~
+
+.. note::
+
+   This approach requires you to have `Git
+   <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__
+   installed.
+
+First, clone our `js-starter repository
+<https://github.com/mongodb-university/js-starter>`__  in your command
+line interface using the following command:
+
+.. tabs::
+
+   .. tab:: SSH
+      :tabid: ssh
+      
+      .. code-block:: none
+
+         git clone git@github.com:mongodb-university/js-starter.git 
+
+   .. tab:: HTTPS
+      :tabid: https
+      
+      .. code-block:: none
+
+         git clone https://github.com/mongodb-university/js-starter.git
+
+Then, navigate into that directory so you can work directly with your
+project's files:
+
+.. code-block:: none
+
+   cd js-starter
+
+At this point, you should have appropriate version of Node.js and NPM
+installed as well as a project directory that contains the dependencies you
```

Comment:
Question: If the person already has all the dependencies installed in the directory, why do they need NPM? Should people going down this path not need NPM?

Reply:
Good catch -- I'll remove NPM as a requirement for cloning the repo